### PR TITLE
Remove PHPcpd

### PR DESCRIPTION
**Pull request by rajeshreeputra**
Fixes: #21

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,8 +17,10 @@
         "php-parallel-lint/php-parallel-lint": "^1.2",
         "phpcompatibility/php-compatibility": "^9.0",
         "phpmd/phpmd": "^2.8",
-        "phpro/grumphp-shim": "^2.0",
-        "sebastian/phpcpd": "^3.0 || ^4.0 || ^5.0 || ^6.0"
+        "phpro/grumphp-shim": "^2.0"
+    },
+    "replace": {
+        "sebastian/phpcpd": "*"
```
Comment (hussainweb) [inline composer.json:23]: Why are you adding this to `replace`?

File: grumphp.yml.dist [hunk 2]
```diff
@@ -18,8 +18,6 @@ grumphp:
         - modules/contrib
         - themes/contrib
         - profiles/contrib
-    phpcpd:
-      directory: ['./web/modules/custom']
```
Comment (hussainweb) [inline grumphp.yml.dist:22]: The problem here is that `phpcpd` is still a useful tool. Can you consider and comment on #12 to still make it available? If yes, then we can keep these lines.

File: composer.json (hunk 1 above)
Comment (rajeshreeputra) [reply composer.json:23]: I think this is not required anymore.

File: grumphp.yml.dist (hunk 2 above)
Comment (rajeshreeputra) [reply grumphp.yml.dist:22]: Will add [phpstan](https://tomasvotruba.com/blog/2021/01/11/how-to-detect-complex-duplicated-methods-with-phpstan) for `copy past detection`.

**Review by hussainweb — APPROVED** [review]
Sounds good. I'll wait for your next PR.

**Pull request merged**